cmd/karavictl/cmd/role_update.go

```diff
@@ -30,8 +30,8 @@ import (
 func NewRoleUpdateCmd() *cobra.Command {
 	roleUpdateCmd := &cobra.Command{
 		Use:   "update",
-		Short: "Update one or more Karavi roles",
-		Long:  `Updates one or more Karavi roles`,
+		Short: "Update the quota of one or more CSM roles",
+		Long:  `Updates the quota of one or more CSM roles`,
```

Comment:
please update in other role commands as well: https://github.com/dell/karavi-authorization/blob/main/cmd/karavictl/cmd/role_create.go#L38

Reply:
Updated.